package.json

```diff
@@ -20,7 +21,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-promise": "^5.1.1",
     "husky": "^7.0.4",
-    "jasmine": "^3.10.0",
+    "jasmine": "^5.0.1",
```

Comment:
I imagine the Node 12 CI test is failing because of a versioning / dependency issue. Try the following steps:
1) Changing the versions of Jasmine and ESLint back to the originals in **package.json**;
2) Delete **node_modules** folder and **package-lock.json** file
3) Re-run `npm i`
4) Stage and commit the changes
5) Push